package.json

```diff
@@ -67,5 +67,21 @@
     "keymirror": "^0.1.1",
     "redux-logger": "^3.0.6",
     "tslint": "^5.13.1"
+  },
```

Comment:
none of the changes in this file should be added.

Reply:
Yes, I tested all the cases using original `package.json` file and it works and I restored this file.
Thanks!